agent/statemanager/state_manager.go

```diff
@@ -79,6 +79,7 @@ const (
 	//   b) Add 'GPUIDs' field to 'apicontainer.Container'
 	//   c) Add 'NvidiaRuntime' field to 'api.task.task'
 	// 20)  Add 'target' field to the Secret struct
+	// 21) Add 'attachmentType' field to 'api.ENIAttachment'
```

Comment:
Comment says `21`. The version is still `20`. What's the mismatch?

Reply:
missed this during rebase. updated data version to 21.